common/recordings.md

```diff
@@ -98,3 +99,51 @@ You can specify a cloud storage location for your recording files in your templa
 ![Destinations](/docs/docs/v2/recording-storage-settings-step6.png)
 
 The above message ensures that your configuration is successful now, and all your recordings will start collecting in your configured destination.
+
+## Storage path for recordings
+
+If a storage destination is not configured for recordings and if you choose to record that room then such recordings are stored for **72 hours** in an internal 100ms bucket. You can access these recordings through [Sessions](https://dashboard.100ms.live/sessions).  
+
+
+![Recording Links](/docs/docs/v2/recording-links-session.png)
+
+
+If a storage destination is not configured for recordings and if you choose to record that room then such recordings are stored for **72 hours** in an internal 100ms bucket. You can access these recordings through [Sessions](https://dashboard.100ms.live/sessions).  
+
+
+![Recording Links](/docs/docs/v2/session-recording-links-1.png)
+
+
+**If a storage destination is configured for recordings, then you can find recording paths in the following webhook responses:**
```

Comment:
this line is confusing. these webhooks will have url even if the "storage destination" is not configured. webhook payload has urls for 100ms bucket if the customer does not define their own storage destination